src/main/kotlin/no/nb/mlt/wls/domain/ports/inbound/MoveItem.kt

```diff
@@ -0,0 +1,21 @@
+package no.nb.mlt.wls.domain.ports.inbound
+
+import no.nb.mlt.wls.domain.model.HostName
+import no.nb.mlt.wls.domain.model.Item
+
+/**
+ * This port is used for handling messages regarding items moving
+ * inside the storage system.
+ * Some examples include handling status messages when crates arrive
+ * at picking stations, or when items return to the
+ * storage systems.
+ * In both cases we want to know where the item went, and if the count changed
+ */
+interface MoveItem {
+    suspend fun moveItem(
+        hostId: String,
+        hostName: HostName,
+        quantity: Double,
+        location: String
```

Comment:
Should check if location is not a blank value. Could probably do that and the above check with annotations, or do a logic check somewhere else in the logic flow, probably service if I were to make the call.

Which reminds me that we should also make sure that when an item is registered the location is set to a null, but only then, cause it's the only case where location can be null, at least so far.

Reply:
With those changes made, we should also have tests to validate that of course.

Reply:
For the latter: Items are registered using `ItemMetadata`, which when mapped to `Item` sets these fields to null. They will only be mapped if you explictly set them. If you'd like I could try writing a comment which describes this intention